curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+To add a title
 
 ---
 
-Placeholder distractor 3
+To change image color
 
 #### --answer--
 
-Placeholder answer
+To provide a text alternative
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a screen reader?
 
 #### --distractors--
 
-Placeholder distractor 1
+A browser for disabled users
 
 ---
 
-Placeholder distractor 2
+A CSS tool
+
 
 ---
 
-Placeholder distractor 3
+A web development library
 
 #### --answer--
 
-Placeholder answer
+A text-to-speech software
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the 'Inspect Element' feature do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property controls text size?
 
 #### --distractors--
 
-Placeholder distractor 1
+text-size
 
 ---
 
-Placeholder distractor 2
+size-font
 
 ---
 
-Placeholder distractor 3
+text-style
 
 #### --answer--
 
-Placeholder answer
+font-size
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the opacity property do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changes text size
 
 ---
 
-Placeholder distractor 2
+Adds a border
 
 ---
 
-Placeholder distractor 3
+Sets the element's position
 
 #### --answer--
 
-Placeholder answer
+Controls element transparency
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide content from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the flex property used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing text color
 
 ---
 
-Placeholder distractor 2
+Adjusting font size
 
 ---
 
-Placeholder distractor 3
+Adding animations
 
 #### --answer--
 
-Placeholder answer
+Creating responsive layouts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool allows you to test CSS live on a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+CodePen
 
 ---
 
-Placeholder distractor 2
+JSFiddle
 
 ---
 
-Placeholder distractor 3
+Live CSS Editor
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute is essential for accessible images?
```

Comment:
This question feels too vague to me. In order to see an image, yes we need a `src` attribute. But for images to be handled properly by a screen-reader and other assistive technologies.